### PR TITLE
change table headers to reflect what is received in response

### LINKED ISSUE #285: Render damage table header as it is
Render damage table header as it is
Since FEMA table as different header "Accept", "Damaged", etc than "Insignificant", "Moderate", etc
The postprocessing script now just return the name instead of DS_0, DS_1 etc
Frontend table need to be updated accordingly
Related to issue https://github.com/IN-CORE/pyincore/issues/488

### PR DESCRIPTION
**Pull request by Rashmil-1999**
Right now when you run it, there will be no visual changes but as soon as the backend is updated to send different keys instead of regular "DS_x" values, the table will reflect what those keys are.

### CODE REVIEW

**Review by longshuicy — APPROVED** [review]

File: src/components/childComponents/table/ResultTable.jsx [hunk 1]
```diff
@@ -111,6 +111,18 @@ export function ResultTable(props) {
 
 	const classes = useStyles();
 
+
+	let tableHeaders = ["Insignificant", "Moderate", "Extensive", "Complete"];
+	let tableKeys = ["DS_0", "DS_1", "DS_2", "DS_3"]
+
+	if (fields && "Insignificant" in fields["by_category"][0]) {
+		tableKeys = ["Insignificant", "Moderate", "Extensive", "Complete"];
```
Comment (longshuicy) [inline src/components/childComponents/table/ResultTable.jsx]: Can we actually just assume using `Object.keys(fields["by_category"][0])` as both the tableHeader and tableKeys..
Also why would need tableHeader and tableKeys? Shouldn't they just be the same?

**Review by longshuicy — CHANGES_REQUESTED** [review]
Sorry I just noticed something. Left you a comment

**Comment by Rashmil-1999** [comment]
I have set it to use just the table headers.... If you run it now you will not see anything as the backend isn't updated. Once we have that updated, I think it should work...

File: src/components/childComponents/graph/SectorShocksBarChart.jsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ const ParseData = (data) => {
 		regions = regions.sort(sortFunc);
 
 		for (const [subKey, value] of regions) {
-			chartData.push([`${key}_${subKey}`, value, colors[i % 5]]);
+			chartData.push([`${key}_${subKey}`, value, colors[i % 4]]);
```
Comment (longshuicy) [inline src/components/childComponents/graph/SectorShocksBarChart.jsx]: Hmm why is this change needed?

File: src/components/childComponents/table/ResultTable.jsx [hunk 3]
```diff
@@ -111,6 +111,13 @@ export function ResultTable(props) {
 
 	const classes = useStyles();
 
+
+	let tableHeaders = ["Insignificant", "Moderate", "Extensive", "Complete"]
+	if (fields && ("Insignificant" in fields["by_category"][0])) {
```
Comment (longshuicy) [inline src/components/childComponents/table/ResultTable.jsx:116]: Sorry for nitpicking, I understand you'd like to set a fallback tableHeaders in 115 which is great.
But then why can't you just unpack what the backend send you using `Object.keys(fields["by_category"])` this way we will have the maximum flexibility.

File: src/components/childComponents/graph/SectorShocksBarChart.jsx (hunk 2 above)
Comment (Rashmil-1999) [reply src/components/childComponents/graph/SectorShocksBarChart.jsx]: wait why does this always happen when I pull and update from develop :(

File: src/components/childComponents/table/ResultTable.jsx (hunk 3 above)
Comment (Rashmil-1999) [reply src/components/childComponents/table/ResultTable.jsx:116]: It has one more entry for category which is why I didn't directly use it... I could slice it though which is why I just checked for the presence of the key. let me do the slice one then

File: src/components/childComponents/graph/SectorShocksBarChart.jsx (hunk 2 above)
Comment (Rashmil-1999) [reply src/components/childComponents/graph/SectorShocksBarChart.jsx]: this is some mistake...

File: src/components/childComponents/table/ResultTable.jsx (hunk 3 above)
Comment (longshuicy) [reply src/components/childComponents/table/ResultTable.jsx:116]: Ah that make sense. Then forget about it haha.
Let's worry about it if there is a workshop 5 :p

**Comment by Rashmil-1999** [comment]
I think everything looks good now...

**Review by longshuicy — APPROVED** [review]

**Pull request merged**